vllm/utils.py

```diff
@@ -101,14 +102,31 @@ def put(self, key: Hashable, value: T) -> None:
         self.cache.move_to_end(key)
         self._remove_old_if_needed()
 
+    def pin(self, key: Hashable) -> None:
```

Comment:
can you add a docstring explaining what "pin" means, to make it easier for future contributors?

Reply:
Added.